dev/dags/dbt/jaffle_shop/fail.txt

```diff
@@ -0,0 +1,77 @@
+adal==1.2.7
```

Comment:
can we please add a comment here to describe what this file means?

Reply:
Removed in 5b84560e845ed6c0f8c24ca38697f08bed9a3339, they were left overs!